orbital-mechanics/Lecture7/Lecture7.ipynb

```diff
@@ -1,386 +1,425 @@
 {
```

Comment:
<p>Again, put equation in its own line for bullet points 1 and 2.</p><p> </p><p>For 3 and 4, shift every rightarrow to its own line.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angadhn/SpacecraftDynamics/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Should I get rid of the L7_10 as I just realised it's not much of an equation, more of a calculation.</p>

Reply:
Nah, it's fine. Don't worry about it.

Reply:
For some reason, I can't see the other comment here but I got a notification. Maybe you deleted it?

Reply:
<p>Ah, yes. I wrote something. Then decided it was self-explanatory and then deleted it.</p>